### PR TITLE
CKS-550 Create lambda directory structure

### PR DESCRIPTION
**Pull request by colinmazu**

### CODE REVIEW

File: terraform/deploy-terraform.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+if [ "$5" = "octo" ]
+  then
+    runningInOctoDeploy=true
+  else
+    runningInOctoDeploy=false
+fi
+
+echo "setting release to $3 and deploying to environment $4"
+releaseNumber=$3
```
Comment (ediblecode) [inline terraform/deploy-terraform.sh:16]: Can you declare all the 5 variables at the top?

File: terraform/deploy-static-to-s3.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+# Usage deploy-static-to-s3.sh <name of s3 bucket> <build number>
+# eg deploy-static-to-s3.sh nice-cks-local-s3-web-hosting 123
+echo "Deploying files to S3 bucket: $1 into folder $2"
```
Comment (ediblecode) [inline terraform/deploy-static-to-s3.sh:4]: Can you create variables for each of the 3 arguments?

File: terraform/deploy-static-to-s3.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+# cache-control: public, max-age=0, must-revalidate
+aws s3 cp ../gatsby/public/ s3://$1/$2 \
```
Comment (ediblecode) [inline terraform/deploy-static-to-s3.sh:7]: Can use a parameter for the gatsby path? e.g.
```suggestion
$gatsbyStaticSitePath=$1
aws s3 cp $gatsbyStaticSitePath s3://$1/$2 \
```

**Review by ediblecode — CHANGES_REQUESTED** [review]

File: terraform/deploy-terraform.sh (hunk 1 above)
Comment (colinmazu) [reply terraform/deploy-terraform.sh]: yep - done

**Review by ediblecode — APPROVED** [review]

**Pull request merged**